src/create_disk.sh

```diff
@@ -214,22 +214,16 @@ case "$arch" in
         -n ${ROOTPN}:0:"${rootfs_size}" -c ${ROOTPN}:root -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4
         ;;
     s390x)
+        set -- -n ${BOOTPN}:0:+384M -c ${BOOTPN}:boot \
```

Comment:
I find it a bit odd to manipulate the global `$@` state here.  Why not a shared array instead?